cogs/server.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, bot):
         embed.add_field(name="Duration:", value=data["roundduration"])
         embed.add_field(name="Mode:", value=data["mode"])
         embed.add_field(name="Players:", value=data["players"])
-        embed.add_field(name="Admins:", value=data["admins"])
+        embed.add_field(name="Staff:", value=data["staff"])
```

Comment:
Don't rename data field. This is inappropriate. It will fail and break things. Do this unless there is according change on API side.

Reply:
I know that this will break it. That's why I put in dev chat request to change this data field in API to. Leaving it "admins" is misleading.

Reply:
https://github.com/Aurorastation/Aurora.3/blob/master/code/modules/world_api/commands/server_query.dm#L23